src/main/java/pl/szczesniak/dominik/whattowatch/movies/domain/MoviesToWatchService.java

```diff
@@ -28,10 +30,31 @@ public void removeMovieFromList(final MovieId movieId, final UserId userId) {
 		repository.removeMovie(movieId, userId);
 	}
 
-	public List<Movie> getList(final UserId userId) {
+	public List<Movie> getMoviesToWatch(final UserId userId) {
 		return repository.findAll(userId);
 	}
 
+	public List<WatchedMovie> getWatchedMovies(final UserId userId) {
+		return watchedRepository.findAllBy(userId);
+	}
+
+	public void moveMovieToWatchedList(final MovieId movieId, final UserId userId) {
+		userCheck(userId);
+		final WatchedMovie watchedMovie = new WatchedMovie(
```

Comment:
Przypomnij mi, żebym Ci pokazał jak to można zrefaktorować żeby to movie miało metodę markAsWatched, albo watch()